recipes/libtiff/all/conanfile.py

```diff
@@ -185,24 +183,25 @@ def package(self):
         self.copy("COPYRIGHT", src=self._source_subfolder, dst="licenses", ignore_case=True, keep_path=False)
         cmake = self._configure_cmake()
         cmake.install()
-        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
 
     def package_info(self):
         self.cpp_info.set_property("cmake_find_mode", "both")
         self.cpp_info.set_property("cmake_file_name", "TIFF")
         self.cpp_info.set_property("cmake_target_name", "TIFF::TIFF")
-        self.cpp_info.set_property("pkg_config_name", "libtiff-{}".format(tools.Version(self.version).major))
+        self.cpp_info.set_property("pkg_config_name", "libtiff-{}".format(Version(self.version).major))
```

Comment:
```suggestion
        self.cpp_info.set_property("pkg_config_name", "libtiff-{}".format(Version(self.version).major))
```

There is no `major` accessor in `conan.tools.scm.Version`, it returns `None`.

Reply:
`major` is a method of `Version`, so it would print e.g. `<bound method Version.major of '3.4'>`

Reply:
It is not, conans.tools.Version != conan.tools.scm.Version ;)

Reply:
I am saying `major` is also available on `conan.tools.scm.Version`, not as a property, but as a method.

Run the following in your terminal:
```sh
python -c "import conan.tools.scm; v = conan.tools.scm.Version('5.4.3.2'); print(f'{v.major=}')"
```

Reply:
Indeed

Reply:
so something like this would work: `Version(self.version).major(fill=False)`

Reply:
I changed it with `version[0]`, that looked simpler to me.